client/asset/dcr/dcr.go

```diff
@@ -3787,12 +3779,12 @@ func (dcr *ExchangeWallet) findRedemptionsInMempool(contractOutpoints []outPoint
 	}
 
 	for _, txHash := range mempoolTxs {
-		tx, err := dcr.wallet.GetRawTransaction(dcr.ctx, txHash)
```

Comment:
Doesn't `getrawtransaction` work for mempool transaction even if `txindex` is not enabled?

Reply:
Yes, for mempool transactions.

Should we leave it in with a note that it can only be used for mempool?

My reasoning is that if `GetTransaction` is good enough for rpc clients with an spv node, then it's good enough for full nodes.

Reply:
Just to note for testing: the dcr **simnet** test harness node alpha _has_ `--txindex`; node beta does _not_ have a tx index.

Reply:
beta wallet is also an --spv wallet so I dont think we have a simnet wallet connecting to beta node